### PR TITLE
Manifestv2

### PR DESCRIPTION
**Pull request by Salamandar**

### CODE REVIEW

**Comment by Salamandar** [comment]
!testme

**Comment by Salamandar** [comment]
!testme

File: doc/PRE_INSTALL_fr.md
Comment (JocelynDelalande) [inline doc/PRE_INSTALL_fr.md:1]: « Connexion unifiée » ?

File: scripts/upgrade [hunk 1]
```diff
@@ -87,9 +87,8 @@ if [ -z "$hashed_password" ] && [ test -f "${install_dir}/venv/bin/activate" ];
 	)
 
 	hashed_password=$(cat $install_dir/key.txt)
-#REMOVEME? 	ynh_secure_remove --file="$install_dir/key.txt"
-#REMOVEME? 	ynh_app_setting_set --app=$app --key=hashed_password --value=$hashed_password
-	ynh_send_readme_to_admin --app_message="../conf/upgrade.msg" --type="upgrade"
+	ynh_secure_remove --file="$install_dir/key.txt"
+ 	ynh_app_setting_set --app=$app --key=hashed_password --value=$hashed_password
```
Comment (JocelynDelalande) [inline scripts/upgrade:83]: btw, why storing the hashed admin password in settings ?

Comment (Salamandar) [reply scripts/upgrade:83]: I'm not sure, that's not my code actually, see https://github.com/YunoHost-Apps/ihatemoney_ynh/commit/b56654d4c1698464e59169985eaf33c6e87a8e16

Comment (Salamandar) [reply scripts/upgrade:83]: Ah, it's to be able to regenerate ihatemoney.cfg on upgrade.

File: doc/PRE_INSTALL_fr.md
Comment (Salamandar) [reply doc/PRE_INSTALL_fr.md:1]: Yeah that's about the (unsupported) SSO.

Comment (JocelynDelalande) [reply doc/PRE_INSTALL_fr.md:1]: My comment was a wording change suggestion. « Connexion unifiée » seems more French than « Login unifié ».

Comment (Salamandar) [reply doc/PRE_INSTALL_fr.md:1]: Ah ! :D

Comment (Salamandar) [reply doc/PRE_INSTALL_fr.md:1]: Done :)

**Comment by ericgaspar** [comment]
!testme

**Comment by Salamandar** [comment]
!testme

**Comment by Salamandar** [comment]
!testme

**Comment by Salamandar** [comment]
!testme

File: doc/POST_UPGRADE.d/4.1.5~ynh3.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
 I Hate Money was successfully upgraded :)
-Please open https://__DOMAIN____PATH_URL__/admin
+Please open https://__DOMAIN____PATH__/admin
 The admin password is: __PASSWORD__
-If you facing an issue or want to improve this app, please open a new issue in this project: https://github.com/YunoHost-Apps/ihatemoney_ynh
\ No newline at end of file
+If you facing an issue or want to improve this app, please open a new issue in this project: https://github.com/YunoHost-Apps/ihatemoney_ynh
```
Comment (alexAubin) [inline doc/POST_UPGRADE.d/4.1.5~ynh3.md]: Hmmm not sure to understand why this is specific to version 4.1.5 ? And also why would you need to display the admin password when upgrading o_O (Maybe there's a good reason but sounds pretty funky)

**Review by alexAubin — APPROVED** [review]
(Approving, though still gotta check the CI)

File: doc/POST_UPGRADE.d/4.1.5~ynh3.md (hunk 2 above)
Comment (Salamandar) [reply doc/POST_UPGRADE.d/4.1.5~ynh3.md]: git log reveals : 

> +* During upgrade from version 4.1.5~ynh3, a new admin password is generated and sent to the root

**Comment by alexAubin** [comment]
(it's good if we can target `testing` instead of `master` tho)

**Comment by Salamandar** [comment]
!testme

**Comment by Salamandar** [comment]
!testme

File: scripts/change_url [hunk 3]
```diff
@@ -84,45 +23,25 @@ ynh_systemd_action --service_name=$app --action="stop"
...
+ynh_change_url_nginx_config
 
 #=================================================
 # SPECIFIC MODIFICATIONS
 #=================================================
 # Setup ihatemoney.cfg
 #=================================================
-
-path_url="$new_path"
+#REMOVEME?
```
Comment (alexAubin) [inline scripts/change_url]:
```suggestion
```

**Review by alexAubin — APPROVED** [review]

File: scripts/change_url (hunk 3 above)
Comment (Salamandar) [reply scripts/change_url]: Thanks :D I'm still fixing the upgrade from the old version.

**Comment by Salamandar** [comment]
!testme

**Pull request merged**